### PR TITLE
fix: Wrong security scan count on security pagination

### PR DESCRIPTION
**Pull request by dhananjay0601**
# Description

Wrong security scan count on security pagination

Fixes https://github.com/devtron-labs/dashboard/issues/1248

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [X] By visiting the security scan page and changing the page size

# Checklist:

* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [X] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/security/SecurityScansTab.tsx [hunk 1]
```diff
@@ -207,10 +211,7 @@ export class SecurityScansTab extends Component<RouteComponentProps<{}>, Securit
   }
 
   changePageSize(newPageSize: number): void {
```
Comment (milindk8) [inline src/components/security/SecurityScansTab.tsx:213]: Search is cleared when page size changes, why ?
we should retain the  search even if user change the page size

File: src/components/security/SecurityScansTab.tsx [hunk 2]
```diff
@@ -191,6 +191,10 @@ export class SecurityScansTab extends Component<RouteComponentProps<{}>, Securit
   }
 
   search() {
+    const searchParams = new URLSearchParams(this.props.location.search)
```
Comment (milindk8) [inline src/components/security/SecurityScansTab.tsx:194]: and when user search offset should be 0

File: src/setupProxy.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ module.exports = function (app) {
     app.use(
         '/orchestrator',
         createProxyMiddleware({
-            target: 'https://demo.devtron.info/',
```
Comment (shivani170) [inline src/setupProxy.js:7]: revert this changes

**Review by vivek-devtron — APPROVED** [review]

**Review by shivani170 — APPROVED** [review]

**Review by shivani170 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]
